micro-rdk/src/esp32/dtls.rs

```diff
@@ -426,24 +510,24 @@ impl SSLContext {
     }
 }
 
-pub struct SSLStream<S> {
+pub(crate) struct SSLStream<S> {
```

Comment:
this is holding a pointer and interacting with it in an unsafe way, is it ok that you don't have a drop implementation for de-allocating the memory behind the pointer?

Reply:
if you are talking about bio_ptr it should be dropped on line 519

Reply:
Ah I missed that, my bad